drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImplTest.java

```diff
@@ -70,13 +68,11 @@
     @Mock
```

Comment:
Not related to your PR: `moduleServiceMock` seems not used. If true, I suggest to remove it.